docs/cloud-adoption/governance/governance-single-team.md

```diff
@@ -1,14 +1,12 @@
 ---
-title: Azure governance design guide
+title: "Enterprise Cloud Adoption: Governance design for a simple workload"
 description: Guidance for configuring Azure governance controls to enable a user to deploy a simple workload
-author: petertay
+author: petertaylor9999
 ---
 
-# Azure governance design guide
+# Enterprise Cloud Adoption: Governance design for a simple workload
 
-The audience for this design guide is the *central IT* persona in your organization. *Central IT* is responsible for designing and implementing your organization's cloud governance architecture. As you learned in the [what is cloud resource governance?](governance-explainer.md) explainer, governance refers to the ongoing process of managing, monitoring, and auditing the use of Azure resources to meet the goals and requirements of your organization.
-
-The goal of this guidance is to help you learn the process of designing your organization's governance architecture by looking at a set of hypothetical governance goals and requirements. Then, we'll discuss how to configure Azure's governance tools to meet them. 
+The goal of this guidance is to help you learn the process for designing a resource governance model in Azure to support a single team and a simple workload.  We'll look at a set of hypothetical governance requirements, then go through several example implementations that satisfy those requirements. 
 
 In the foundational adoption stage, our goal is to deploy a simple workload to Azure. This results in the following requirements:
```

Comment:
Are we still using the "adoption stage" model? With the restructuring, this is the first time it's been introduced (I think)